hw/ip/adc_ctrl/dv/ast_adc_agent/ast_adc_agent.core

```diff
@@ -0,0 +1,34 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:dv:ast_adc_agent:0.1"
+description: "AST_ADC DV UVM agent"
+filesets:
+  files_rtl:
+    depend:
+      - lowrisc:ip:adc_ctrl
```

Comment:
What exactly does the agent need from the design? As such, agents need to be reusable and must refrain from referencing design details directly.

Reply:
is there are specific parameters need, let us know we can break out a package core file.